src/helpers.php

```diff
@@ -96,18 +97,56 @@ function graphql(
     string $source,
     $rootValue = null,
     $contextValue = null,
-    $variableValues = null,
+    $variableValues = [],
     $operationName = null,
     callable $fieldResolver = null
 ): ExecutionResult {
     /** @noinspection PhpParamsInspection */
-    return Execution::execute(
-        $schema,
-        parse($source),
-        $rootValue,
-        $contextValue,
-        $variableValues,
-        $operationName,
-        $fieldResolver
-    );
+    return GraphQL::getInstance()
```

Comment:
This is supposed to be `GraphQL::get(...)`.

Reply:
Lol, I copied your code above...